app/helpers/interruption_screen_helper.rb

```diff
@@ -7,6 +7,8 @@ def display_informative_text(informative_text, lettings_log)
       value = if argument["label"]
                 pre_casing_value = lettings_log.form.get_question(argument["key"], lettings_log).answer_label(lettings_log)
                 pre_casing_value.downcase
+              elsif argument["currency"]
+                ["£", ActionController::Base.helpers.number_to_currency(lettings_log.public_send(argument["key"]), delimiter: ",", format: "%n")].join("")
```

Comment:
we should discuss this so I understand properly what it's doing.

also default argument of join is "" so we can remove that

Reply:
also while discussing this we should discuss "label"